avro/profile.json

```diff
@@ -0,0 +1,118 @@
+{
+  "name":"profile",
+  "type":"record",
+  "namespace": "dto",
+  "doc":"Schema for cosmos api container 'profiles'",
+  "fields":[
+    {
+      "name": "id",
+      "type": "string",
+      "default": "undefined"
+    },
+    {
+      "name":"version",
+      "type":"int",
+      "default":0
+    },
+    {
+      "name": "timestamp",
+      "type": "long",
+      "logicalType": "timestamp-millis",
+      "default": 0
+    },
+    {
+      "name":"blockedInboxOrChannels",
+      "doc": "Contains blocked channels per each service. This info remains for backward compatibility only, it has been replaced with service-preferences data",
+      "type":{
+        "type": "map",
+        "values":{
+          "type": "array",
+          "items": {
+            "type":"enum",
+            "name":"BlockedInboxOrChannelEnum",
+            "symbols":[
+              "EMAIL",
+              "WEBHOOK",
+              "INBOX"
+            ]
+          }
+        }
+      },
+      "default" : {}
+    },
+    {
+      "name":"userPDVId",
+      "type": "string",
+      "default": "undefined"
```

Comment:
this should never been set to undefined default.

Reply:
done in https://github.com/pagopa/io-functions-elt/pull/81/commits/9c449a0343484f07576dcfbc160d613e0651c285